modules/041-linstor/templates/nodegroupconfiguration-drbd-install-altlinux-like.yaml

```diff
@@ -0,0 +1,127 @@
+apiVersion: deckhouse.io/v1alpha1
+kind: NodeGroupConfiguration
+metadata:
+  name: drbd-install-altlinux.sh
+  {{- include "helm_lib_module_labels" (list .) | nindent 2 }}
+spec:
+  weight: 98
+  nodeGroups: ["*"]
+  bundles: ["altlinux"]
+  content: |
+    # Copyright 2023 Flant JSC
+    #
+    # Licensed under the Apache License, Version 2.0 (the "License");
+    # you may not use this file except in compliance with the License.
+    # You may obtain a copy of the License at
+    #
+    #     http://www.apache.org/licenses/LICENSE-2.0
+    #
+    # Unless required by applicable law or agreed to in writing, software
+    # distributed under the License is distributed on an "AS IS" BASIS,
+    # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    # See the License for the specific language governing permissions and
+    # limitations under the License.
+
+    kubeconfig="/etc/kubernetes/kubelet.conf"
+
+    {{- if dig "dataNodes" "nodeSelector" false .Values.linstor }}
+    need_drbd=$(bb-kubectl --kubeconfig $kubeconfig  get node "$(hostname -s)" -o json | jq -c '.metadata.labels | contains({{ .Values.linstor.dataNodes.nodeSelector | toJson }})')
+    {{- else }}
+    need_drbd="true"
+    {{- end }}
+
+    bb-log-info "we need drbd on node: "$need_drbd
+
+    if [ $need_drbd == "true" ]; then
+      CLUSTER_DNS="{{ .Values.global.discovery.clusterDNSAddress }}"
+      SPAAS_FQDN="spaas.d8-linstor.svc.{{ .Values.global.discovery.clusterDomain }}"
+
+      if [ -e "/proc/drbd" ]; then
+        # DRBD check version
+
+        current_version="$(cat /proc/drbd | grep 'version:' | grep -oE '[0-9]+\.[0-9]+\.[0-9]+')"
+        desired_version="{{ $.Values.linstor.internal.drbdVersion }}"
+
+        # We expect the loaded DRBD module to be version 9.
+        # If version 8 is loaded, it means that for some reason, the in-tree kernel module has been automatically loaded.
+        # (For example, this can happen due to drbd-utils installed on the host, which should not occur in standard scenarios).
+        # We are only interested in the version 9 loaded by our helper script, so unload module and wait until it done.
+        if [[ ! $current_version =~ ^9.* ]]; then
+          rmmod drbd_transport_rdma || true
+          rmmod drbd_transport_tcp || true
+          rmmod drbd || true
+        fi
+
+        if [ "${current_version}" == "${desired_version}" ]; then
+          bb-log-info "Actual drbd version loaded"
+          exit 0
+        else
+          bb-log-info "Non-actual version of drbd (now "$current_version", desired "$desired_version"), setting reboot flag"
+          bb-flag-set reboot
+        fi
+      fi
+
+      bb-rp-install "drbd924:{{ .Values.global.modulesImages.digests.registrypackages.drbd924 }}"
+      # Here we install headers for further usage. It can be different from current kernel version! So further we check, if we need new kernel version.
```

Comment:
```suggestion
      # Here we install headers for further usage. It can be different from current kernel version! So further we check, if we need new kernel version. It is altlinux feature.
```